certbot/CHANGELOG.md

```diff
@@ -2794,7 +2794,7 @@ by default, these files can be used to control what parameters are
 used when renewing a specific certificate.
```

Comment:
I don't think we should change any past entries in the CHANGELOG, they are immutable. 

In any case, GitHub has a redirect for these URLs so it's not imperative to make this change.